docker/assertj/src/main/java/com/github/dockerjava/assertions/DockerJavaAssert.java

```diff
@@ -31,6 +35,12 @@ public ContainerAssert container(String name) {
       return new ContainerAssert(container);
    }
 
+   public CubeOutputAssert withContainer(String name) {
+         String[] cmd = {"bash", "-c", "for i in $(ps -axo pid --no-headers); do echo `ps -p $i -o comm=`; done"};
```

Comment:
Well I see that actually it gets `"USER","PID","%CPU","%MEM","VSZ","RSS","TTY","STAT","START","TIME","COMMAND"`

Could we parse the `COMMAND` to see the exact name?

I think that if this is not enough for us, maybe we should drop the support because of `bash`thing. Or we just release with this limitation.